config/daemon/kustomization.yaml

```diff
@@ -0,0 +1,4 @@
+resources:
+- namespace.yaml
```

Comment:
kustomize doesnt like when you define the namespace again:

`Error: namespace transformation produces ID conflict: [{"apiVersion":"v1","kind":"Namespace","metadata":{"labels":{"control-plane":"controller-manager"},"name":"ingress-node-firewall-system"}}{nsfx:false,beh:unspecified} {"apiVersion":"v1","kind":"Namespace","metadata":{"name":"ingress-node-firewall-system"}}{nsfx:false,beh:unspecified}]
error: no objects passed to apply
`

I had to comment this line out to proceed.

Reply:
Interesting ... so when I run `make install`, I need the namespace .. but with `make deploy`, it fails ...